bufferserver/src/main/java/com/datatorrent/bufferserver/internal/LogicalNode.java

```diff
@@ -346,7 +346,9 @@ public void boot(EventLoop eventloop)
   @Override
   public String toString()
   {
-    return "LogicalNode{" + "upstream=" + upstream + ", group=" + group + ", partitions=" + partitions + ", iterator=" + iterator + '}';
+    return getClass().getName() + "@" + Integer.toHexString(hashCode()) +
```

Comment:
To avoid extra processing done by getSimpleName(), I hardcoded name to be LogicalNode. There are classes that extend LogicalNode, so this should be OK as well.